config/log4j-rolling.properties

```diff
@@ -39,6 +39,12 @@ log4j.appender.kafka.MaxFileSize=10MB
 log4j.appender.kafka.MaxBackupIndex=5
 log4j.appender.kafka.append=true
 
+log4j.appender.kafka_appender=org.apache.kafka.log4jappender.KafkaLog4jAppender
```

Comment:
How come these configs were added into `log4j-rolling.properties` rather than `log4j.properties`? I thought the latter is the default?

Reply:
Nope these are the server configs. I think log4j.properties is what the cli uses.